src/components/navigation/menu/PhoneMenu/ToggleButton.js

```diff
@@ -32,9 +32,9 @@ const StyledToggleButton = styled(ToggleButton)`
   right: 30px;
   top: 18px;
   z-index: 9991;
-
+  ${FONT_FAMILY}
```

Comment:
@shakersinthedark you could use the font-family from the design system now. The font-family is set in the Box, If you styled(Box)  (which I believe all our components should come from the Box or a component that extends the Box) then you've got the font-family already.

Reply:
I don't know how to do this so I think we should refactor together when you're free next.